src/procedures/procedures.11tydata.yaml

```diff
@@ -319,6 +319,26 @@ procedureReassembleCase:
           <li>Insert the 4 thumb screws.</li>
           <li class="warning">Do not over-tighten the screws.</li>
         </ul>
+procedureInstallSWNVME:
+  title: Almost done!
+  steps:
+    - title: Finished assembly!
+      image: yellow-assembled.webp
+      content: |
+        <ul>
+          <li>Congratulations! You've completed the assembly.</li>
+          <li>You can now start <a href="/install-software-nvme" target="_blank">installing the Home Assistant Software</a></li>
```

Comment:
This should not be `target="_blank"`, since the user is expected to continue there.

Can we also have a link somewhere to directly go to this guide? Otherwise people need to click through the whole selection process, scroll all the way down, and click this link to go to this guide. E.g. by adding a link in the "Choose your configuration" page of the Kits variant.